deploy/image-sync/CESNET-MCC.tfvars

```diff
@@ -0,0 +1,8 @@
+# Network
+net_id = "5b8fbb48-461d-4907-ac79-7f7b2ebd17f9"
+
+# Flavor: standard.medium   2cores/4GB RAM
+flavor_id = "4c153ce3-a163-4668-baa7-2cbcb57e2dd8"
+
+# Image: ubuntu bionic
+image_id = "e8d75fc1-ac32-4851-90b5-b4c925e9e6f8"
```

Comment:
Are we using `cloud.egi.eu` VO for this? I don't think `CESNET-MCC` is part of this VO?
Anyway, we shouldn't use `ubuntu bionic`?

Reply:
Yes, this is a left over, removed